elks/arch/i86/drivers/char/crtc-6845.h

```diff
@@ -0,0 +1,55 @@
+#define OT_MDA 0
+#define OT_CGA 1
+#define OT_EGA 2
+#define OT_VGA 3
+
+#define N_DEVICETYPES 2
+
+#ifdef DEBUG
+static const char *type_string[] = {
+    "MDA",
+    "CGA",
+    "EGA",
+    "VGA",
+};
+#endif
+
+#if defined(CONFIG_CONSOLE_DUAL)
+struct hw_params {
```

Comment:
I prefer that structures stay defined regardless of config option. While seemingly useless, this allows for other additions to signal an error should they conflict with another option, even though that option is not selected. So this gate can move to line 29.